src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
I pretty much agree, and that looks like a solid argument against `originalEvent: {}`. So now we're left with:
* `originalEvent: new jQuery.Event()`
* `originalEvent: null`
* `e.originalEvent = undefined`
* `delete e.originalEvent`

All are compatible with a873558436383beea7a05fd07db7070a30420100, and the first is compatible with existing documentation.